src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -4950,7 +4950,7 @@ def test_aks_update_with_workload_identity(self, resource_group, resource_group_
 
         disable_cmd = ' '.join([
             'aks', 'update', '--resource-group={resource_group}', '--name={name}',
-            '--enable-workload-identity', 'False',
+            '--disable-workload-identity',
```

Comment:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=78963033&view=results) to validate the change.